### PR TITLE
Fix: New dragged widgets get inserted to the bottom [ED-6170]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/utils/jquery-html5-dnd.js [hunk 1]
```diff
@@ -353,15 +354,27 @@
...
+			}
+
+			const dragged = elementor.channels.panelElements.request( 'element:selected' )?.model.attributes;
+
+			settings.getDropContainer().view.createElementFromModel(
+				{ elType: dragged.elType, widgetType: dragged.widgetType, custom: dragged.custom },
+				{
+					at: settings.getDropIndex( currentSide, event ),
+				}
+			);
```
Comment (StyleShit) [inline assets/dev/js/editor/utils/jquery-html5-dnd.js:374]: Same as in the Container branch

File: assets/dev/js/editor/utils/jquery-html5-dnd.js [hunk 2]
```diff
@@ -353,15 +354,27 @@
...
+
+			if ( settings.onAfterDrop ) {
+				settings.onAfterDrop( currentSide, event );
```
Comment (StyleShit) [inline assets/dev/js/editor/utils/jquery-html5-dnd.js:377]: Newly added code

File: assets/dev/js/editor/views/add-section/inline.js [hunk 3]
```diff
@@ -13,6 +13,12 @@ class AddSectionView extends BaseAddSectionView {
 		} );
 	}
 
+	onAfterDrop() {
+		super.onAfterDrop();
+
+		this.destroy();
```
Comment (matipojo) [inline assets/dev/js/editor/views/add-section/inline.js:19]: How has it worked until now?

File: assets/dev/js/editor/views/base-container.js [hunk 4]
```diff
@@ -127,7 +127,7 @@ module.exports = Marionette.CompositeView.extend( {
 				container,
 				columns: Number( ! containerExperiment ),
 				options: {
-					at: this.getOption( 'at' ),
+					at: options.at,
```
Comment (matipojo) [inline assets/dev/js/editor/views/base-container.js:130]: How has it worked until now?

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**